src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.nav_menu {
+  display: flex;
+  justify-content: flex-start;
+  text-transform: uppercase;
+  list-style: none;
```

Comment:
Avoid using tag names for styling. It's better to style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. In this case, you can move `list-style: none;` to `.nav_menu`.